packages/tocco-ui/src/FormattedValue/typeFormatters/TimeFormatter.js

```diff
@@ -10,15 +12,24 @@ const TimeFormatter = props => {
 
   const date = new Date(2000, 1, 1, hours, minutes, seconds, milliSeconds)
 
+  const twoDigits = n => { return String('00' + n).slice(-2) }
```

Comment:
Please do the following: 
- Add the newest version of moment js (yarn add moment)
- use moment js format to get iso format
    const timeIso = moment(date).format(moment.HTML5_FMT.TIME_MS)